repo/meta/schema/v3-repo-schema.json

```diff
@@ -65,6 +113,49 @@
           },
           "additionalProperties": false
         },
+        "cli": {
+            "additionalProperties": false,
+            "properties": {
+                "binaries": {
+                    "additionalProperties": false,
+                    "minProperties": 1,
+                    "properties": {
+                        "darwin": {
+                            "additionalProperties": false,
+                            "properties": {
+                                "x86-64": {
+                                    "$ref": "#/definitions/cliInfo"
+                                }
+                            },
+                            "type": "object"
```

Comment:
Can we mark `x86-64` as required since `additionalProperties` is set to false? and the whole `darwin` property is optional?